README.md

```diff
@@ -11,6 +11,31 @@ issues, thanks to the language's guarantees.
 
 ## Usage
 
+The [user guide][UG] provides an overview of everything available in the 
+project. It can be generated offline using **mdbook**: 
+
+```shell
+# Serve the doc on a local server
+mdbook serve --open -d ../target/doc/ honeycomb-guide/ &
+cargo doc --all --no-deps
+
+# Kill the local server
+kill $(pidof mdbook) 
+
+# Without pidof
+kill $(ps -e | awk '/mdbook/ {print $1}')
+```
```

Comment:
Do you think it is useful?

Reply:
I am reluctant to using a command to run a process independently and leave it as is, without instructions to shut it down. I'll leave it in the README in a separated code bloc and I'll remove it from other instances.